pvlib/spectrum/spectrl2.py

```diff
@@ -227,7 +227,7 @@ def spectrl2(apparent_zenith, aoi, surface_tilt, ground_albedo,
 
     Returns
     -------
-    spectra : dict
+    : dict
```

Comment:
Rather than deleting the problematic name as @cwhanse suggested, what about renaming it to something better?  `spectra_components`, maybe?